pages/common/badblocks.md

```diff
@@ -14,3 +14,11 @@
 - Search an unmounted disk for bad blocks with a destructive write test:
 
 `sudo badblocks -w {{/dev/sda}}`
+
+- Search an unmounted disk in desctructive mode and output found blocks to file:
+
+`sudo badblocks -o {{badblocks.txt}} -w {{/dev/sda}}`
+
+- Search an unmounted disk in desctructive mode with improved speed using 4K block size and 64K block count:
```

Comment:
4K and 64K correspond to 4096 and 65536 right under? If that's the case, then those numbers should be put in inside braces such as `{{4096}}`. Perhaps `with improved speed` is unnecessary.

Reply:
Yes, that is correct. I'll add the brackets around those options. I was trying to say something as to why the use of those options

Reply:
Is the notation `4k` not supported?